terraform/modules/azdo_ubuntuagent/main.tf

```diff
@@ -1,24 +1,24 @@
 
 resource "azurerm_network_interface" "VM" {
   name                      = "${var.vm_name}-nic"
-  location                  = "${var.AZURERM_RESOURCE_GROUP_MAIN_LOCATION}"
-  resource_group_name       = "${var.AZURERM_RESOURCE_GROUP_MAIN_NAME}"
-  network_security_group_id = "${var.AZURERM_NETWORK_SECURITY_GROUP_MAIN_ID}"
+  location                  = var.AZURE_REGION
+  resource_group_name       = var.AZURERM_RESOURCE_GROUP_MAIN_NAME
+  network_security_group_id = var.AZURERM_NETWORK_SECURITY_GROUP_MAIN_ID
 
   ip_configuration {
     name                          = "configuration"
-    subnet_id                     = "${var.AZURERM_SUBNET_ID}"
+    subnet_id                     = var.AZURERM_SUBNET_ID
     private_ip_address_allocation = "Dynamic"
   }
-  tags = "${merge(var.TAGS, { "ACCOUNT" = "${var.VSTS_ACCOUNT}", "RUN_DATE" = "${var.run_date}" })}"
+  tags = merge(var.TAGS, { "ACCOUNT" = "${var.VSTS_ACCOUNT}", "RUN_DATE" = "${var.run_date}" })
 }
 
 resource "azurerm_virtual_machine" "VM" {
-  name                             = "${var.vm_name}"
-  location                         = "${var.AZURERM_RESOURCE_GROUP_MAIN_LOCATION}"
-  resource_group_name              = "${var.AZURERM_RESOURCE_GROUP_MAIN_NAME}"
+  name                             = var.vm_name
+  location                         = var.AZURE_REGION
```

Comment:
Is this an intended change?

Reply:
Was correcting a find and replace using the other module